lib.php

```diff
@@ -1,4 +1,26 @@
 <?php
 
-    //define('MOODLE_INTERNAL') || die();
+    defined('MOODLE_INTERNAL') || die();
 
+    /**
+     * @param navigation_node $navigation
```

Comment:
Might be nice to finish this PHPDoc block https://docs.phpdoc.org/latest/references/phpdoc/basic-syntax.html#what-is-a-docblock